### PR TITLE
Improve multiple UI elements

### LINKED ISSUE #305: increase clickable area range in process list icons

### LINKED ISSUE #315: error color should be consistent
and now they are orange&red
Clipboard - November 7, 2019 12_34 PM

set the error color to be: "#FF9A4D"
background color for nodes: "#FCDCCA"

### LINKED ISSUE #316: Moving edges breaks diagram
**Describe the bug**
Moving edge end from one end to another seems to be working, but diagram is broken after

**To Reproduce**
on demo.nussknacker.io

**Screenshots**

Screencast-from-07-11-19-13_33_59

### LINKED ISSUE #332: fields in node edit modal that are not editable should look not editable

### PR DESCRIPTION
**Pull request by PPiedel**
This fixes #315
This fixes #332 
This fixes #305 
This fixes #316

### CODE REVIEW

**Comment by witekw** [comment]
I think background color of "error" nodes should not be overwritten by "selection" color.

File: ui/client/components/graph/Graph.js [hunk 1]
```diff
@@ -186,13 +186,29 @@ class Graph extends React.Component {
...
+        this.disconnectPreviousEdge(c.model.id);
         this.props.actions.nodesConnected(
           c.sourceView.model.attributes.nodeData,
           c.targetView.model.attributes.nodeData
         )
       })
   }
 
+  disconnectPreviousEdge = (previousEdge) => {
+    const nodeIds = this.extractNodeIds(previousEdge);
```
Comment (witekw) [inline ui/client/components/graph/Graph.js]: Why not unpack the array and show what is going on?

Comment (PPiedel) [reply ui/client/components/graph/Graph.js]: Done

**Comment by PPiedel** [comment]
> I think background color of "error" nodes should not be overwritten by "selection" color.

We've got issue for that: https://github.com/TouK/nussknacker/issues/307

File: ui/client/package.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
   "private": true,
   "scripts": {
     "start": "PROXY_API_DOMAIN=http://localhost:8081/api webpack-dev-server",
-    "start-backend-demo": "PROXY_API_DOMAIN=https://demo.nussknacker.io/api webpack-dev-server",
+    "start-backend-demo": "PROXY_API_DOMAIN=https://staging.nussknacker.io/api webpack-dev-server",
```
Comment (lciolecki) [inline ui/client/package.json]: Change url or backend-demo to backend-stage :)

**Pull request merged**